glide.lock

```diff
@@ -0,0 +1,51 @@
+hash: db55a031aaa2f352fa5e9e4fda871039afb80e383a57fc77e4b35114d47cca8a
```

Comment:
Yes, we definitely check in the lock file. Glide allows developers to specify version ranges in the main glide.yaml file. It then automatically generates the glide.lock file. You can then rebuild an exact image of the last known good build using `glide install`. That reads the lock file and installs exactly the referenced versions. (`glide up`, in contrast, will re-scan the code and see if it can update things and still match the version constraints specified.)